### PR TITLE
(SIMP-2256) Bring in new modules

### PR DESCRIPTION
**Pull request by jeefberkey**
- Added new simplib modules to Puppetfile
- Updated simp_classes to reflect the new changes

SIMP-2236 #close
SIMP-2256 #close
SIMP-1679 #comment updated simp-core

### CODE REVIEW

File: src/assets/simp-environment/environments/simp/manifests/site.pp [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # Place Hiera customizations based on this variable in hieradata/hostgroups/${::hostgroup}.yaml
 #
 # Example hostgroup delcaration using a regex match on the hostname:
-#   if $::fqdn =~ /ws\d+\.${::domain}/ {
+#   if $::fqdn =~ /ws\d+\.<domain>/ {
```
Comment (trevor-vaughan) [inline src/assets/simp-environment/environments/simp/manifests/site.pp]: Does this actually work?

File: src/assets/simp-environment/environments/simp/hieradata/simp_classes.yaml [hunk 2]
```diff
@@ -11,68 +11,62 @@
...
-#   * http://docs.puppetlabs.com/hiera/1/complete_example.html       #
+#   * http://docs.puppetlabs.com/hiera/latest/puppet.html            #
+#   * http://docs.puppetlabs.com/hiera/latest/complete_example.html  #
 #                                                                    #
 ######################################################################
 
 classes:
-  - 'aide'
-  - 'auditd'
+  - '::aide'
```
Comment (trevor-vaughan) [inline src/assets/simp-environment/environments/simp/hieradata/simp_classes.yaml]: Do these work scoped in Hiera? Also, Puppet 4 shouldn't need to be scoped any more but we still do it in the modules for clarity.

Comment (jeefberkey) [inline src/assets/simp-environment/environments/simp/hieradata/simp_classes.yaml]: They do work, and I thought the scoping was needed. Will fix later.

File: src/assets/simp-environment/environments/simp/manifests/site.pp (hunk 1 above)
Comment (jeefberkey) [inline src/assets/simp-environment/environments/simp/manifests/site.pp]: It didn't work before, the // matcher didn't support interpolation.

**Pull request merged**